bridge/opentracing/src/ot_otel_bridge/tracer.py

```diff
@@ -0,0 +1,120 @@
+"""
+OpenTracing bridge: an OpenTracing tracer implementation using the
+OpenTelemetry API.
+"""
+
+import time
+
+import opentracing
+from basictracer import BasicTracer
+from basictracer.context import SpanContext
+from basictracer.text_propagator import TextPropagator
+from basictracer.util import generate_id
+from opentracing.propagation import Format
+
+from .span import BridgeSpan
+
+
+def tracer(**kwargs):  # pylint: disable=unused-argument
+    return _BridgeTracer()
```

Comment:
Shouldn't the kwargs be passed here?

Reply:
But I don't have a kwargs in the `__init__` below. The `_LightstepTracer` class followed this model.